src/Statement.php

```diff
@@ -266,10 +266,14 @@ public function getAuthority()
      * occurred.
      *
      * @return \DateTime The timestamp
+     *
+     * @deprecated since 1.1.1, to be removed in 3.0
      */
     public function getTimestamp()
     {
-        return $this->created;
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.1.1 and will be removed in 3.0. Use "%s::getCreated()" method instead.', __METHOD__, __CLASS__), E_USER_DEPRECATED);
+
+        return $this->getCreated();
```

Comment:
Let's not change this one. There's not really a need for an additional method call here.